pandas/tests/indexes/common.py

```diff
@@ -850,12 +850,9 @@ def test_equals_op(self):
             tm.assert_series_equal(series_a == item, Series(expected3))
 
     def test_numpy_ufuncs(self):
-        # test ufuncs of numpy 1.9.2. see:
+        # test ufuncs of numpy, see:
```

Comment:
should the version number be removed or updated?

Reply:
Ufuncs are here to stay now, no? That's why I removed it.

Unless you mean that the set of ufuncs is related to the version number?